content/en/continuous_integration/_index.md

```diff
@@ -22,6 +22,7 @@ further_reading:
 cascade:
     algolia:
         rank: 70
+        tags: ["ci/cd", "continuous integration"]
```

Comment:
```suggestion
        tags: ['ci/cd', 'continuous integration']
```

Reply:
Not sure if the type of quote matters, but the Search 2.0 internal doc shows single quotes and I'm not finding any examples of double quotes being used for this in the docs repo